.github/workflows/pre_release.yaml

```diff
@@ -0,0 +1,105 @@
+name: Create a pre-release
+
+on:
+    # Push to master will deploy a beta version
+    push:
+        branches:
+            - master
+            # This will allow releasing beta versions from non-master releases.
+            # TODO: We need to release this version with other tags than beta. It can mess up the beta version as it will override it with older code.
```

Comment:
What does this mean exactly?

Reply:
no idea honestly, that was already in the current version :smile:

Reply:
If it makes no sense to nobody (I also have no idea, feel free to check git blame), let's just drop that comment right ahead.

Reply:
I'd also remove the line that follows it :slightly_smiling_face: